docs/compatibility.md

```diff
@@ -24,3 +24,18 @@ With the monkey patch disabled, use `render_component` (or  `render_component_to
 ## Bridgetown (Static Site Generator)
 
 [Bridgetown](https://www.bridgetownrb.com/) supports ViewComponent via an experimental shim provided by the [bridgetown-view-component gem](https://github.com/bridgetownrb/bridgetown-view-component). More information available [here](https://www.bridgetownrb.com/docs/components/ruby#need-compatibility-with-rails-try-viewcomponent-experimental).
+
+## ActionText Support
+
+Trying to use `rich_text_area` via ActionText out of the box with ViewComponent will result in this error:
+```
+undefined method `rich_text_area_tag'
+```
+
+This is because ViewComponent does not have access to the helpers it needs via ActionText. To avoid this, add the following to your component (or base component):
+
+```
+delegate :rich_text_area_tag, to: :helpers
+```
+
+*_This is the simplest route. Adding the helpers directly via `include ActionText::Engine.helpers` leads to several more errors and requires additional configuration that is not necessary._
```

Comment:
Looks great but I'm wondering if it's worth mentioning the other method without saying what it is or linking to it? Imo I believe this to be the "right" way to fix this and it doesn't just pertain to ActionText!

[Here's a great example](https://www.bridgetownrb.com/docs/components/ruby#helpers-1)

Reply:
Good call! I went ahead and removed it.